### ISSUE TITLE
3 panels 64x32

### ISSUE THREAD

**Issue by albinholmqvist** [PPUC/ZeDMD#57]
Hi, is it possible to make the image stretch to 3 panels? (3x 64x32 px)

Or change the resolution??
Thanks in advance!

**Comment by mkalkbrenner** [comment]
Yes, it is possible. But it requires code changes to support a resolution of 192x32:
- centering
- scaling

Especially the scaling won't be straight forward. At the moment it is implemented as up or down scaling that keeps the original aspect ratio. Stretching isn't implemented.

A better way might be to only add support for a native resolution of 192x32 and to only center images of smaller resolutions.
The clients like batocera or libdmdutil or dmdext or dmd-play should then send images in 192x32 to avoid stretching on the device.
But for sure, if someone provides a PR for stretching, I would be happy to review it.

Another option might be a kind of ambilight shown on the left and on the right.
